content/syllabus-written-assignments/file-directory-naming/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+content_type: project
+flavours:
+  - java
+  - javascript
+  - markdown
+  - python
+prerequisites:
+  hard:
+    - topics/file-and-directory-naming/general
+protect_main_branch: false
+ready: true
+submission_type: repo
+title: File and Directory Naming
+---
+
+Please answer the following questions in markdown files and submit following the instructions below.
+
+## Questions
+
+1. Here are some directory names that follow different naming conventions: Foo_Bar, foo_bar, FooBar, main, fooBar. Which of the directory names follows the convention used in your language?
+
+2. Here are some file names that follow different naming conventions: MyFile, my_file, My_File. Which of the file names follows the convention used in your language?
+
+3. Here are function names that follow different naming conventions: myFunction, my_function, MyFunction. Which of the function names follows the convention used in your language?
```

Comment:
Add some crazy ones like My_function My_Function. People do strange mix-and-match like that often enough that it's worth discouraging